### PR TITLE
Refined: Image functionality on resume page 

### LINKED ISSUE #68: Screen is getting blanked while uploading image
**Describe the bug**
In resume page if we try to upload different documents or videos rather then images screen is getting blanked so i think it should display an error message like only jpg,jpeg,png are allowed something like that.

**Screenshots**

https://user-images.githubusercontent.com/68184118/234846101-17f4abba-b934-4a53-9376-b8119a333eb3.mp4

**Additional context**

### PR DESCRIPTION
**Pull request by 777Prabhjot**
- [x] closes #68 #69 

Hey i have implemented add/ remove image functionality for resume template please check it out

### CODE REVIEW

File: src/components/ResumePage/element/BuildSteps/Education.js [hunk 1]
```diff
@@ -106,6 +111,14 @@ const Education = () => {
...
                 </FormControl>
               </HStack>
+              {educationList[0].id !== education.id && <Button
+                rightIcon={<MdDelete />}
+                onClick={() => deleteEducation(education.id)}
+                mt={3}
+                colorScheme={"red"}
+              >
+                Delete
+              </Button>}
```
[the remark this answers is not in the document]
Comment (777Prabhjot) [reply src/components/ResumePage/element/BuildSteps/Education.js:121]: @ShashwatAgrawal20 yes why not

**Comment by ShashwatAgrawal20** [comment]
@777Prabhjot LGTM

Need some tests, keeping it open for some time.

**Comment by ShashwatAgrawal20** [comment]
@Khizarshah01

**Review by Khizarshah01 — COMMENTED** [review]
@ShashwatAgrawal20 yash the new features working properly.

**Comment by ShashwatAgrawal20** [comment]
@777Prabhjot didn't see the **delete button** in the **education section.**

**Pull request merged**